docs/user-guide/styling-your-docs.md

```diff
@@ -54,21 +58,28 @@ customizations, including overriding templates, you will need to use the theme
 
 ### Using the docs_dir
 
-The [extra_css] and [extra_javascript] configuration options can be used to
-make tweaks and customizations to existing themes. To use these, you simply
-need to include either CSS or JavaScript files within your [documentation
-directory].
+The [extra_css] and [extra_javascript] configuration options can be added to
+`mkdocs.yml` to add additional CSS and Javascript files to existing themes.
+Include CSS or Javascript files within `docs_dir` (`docs/` by default) and
```

Comment:
Why are we removing the link here? Wouldn't it be better to leave the link to `docs_dir` rather than add that the default is `docs/`. 

A lot of your confusion seems to stem from contradictory documentation which was introduced when changes were made. By keeping things documented in one location (and linking to is from elsewhere) we can avoid that in the future.

A good rule of thumb I have been trying to follow is whenever a new setting is introduced within a section, link to that setting on the configuration page. That way, users can easily discover what that setting is for, what valid values it accepts and what it defaults to.

True, the docs don't do that everywhere, but every section I'v recently edited does. Slowly the docs have been getting better in that regard.